### PR TITLE
Create placeholder loading page

### LINKED ISSUE #57748: Plugin Details: Create placeholder loading page
Update the Plugin Details page to show a placeholder (gray boxes) when the plugin information still loading.

### PR DESCRIPTION
**Pull request by gcsecsey**
#### Changes proposed in this Pull Request

Show placeholders to the plugin details page while data is fetched.

#### Testing instructions

- Go to plugins page
- Select a plugin
- Check that the loading state has a design matching below:

| |Before | After|
|---|-------|------|
| Desktop | image | image |
| Mobile | image| image |

Fixes #57748

### CODE REVIEW

**Review by WBerredo — DISMISSED** [review]
I would suggest also adding `user-select: none` to placeholder css, as currently the user can still select and display the reticences inside. Demo:

Screen Shot 2021-11-16 at 14 49 19

File: client/my-sites/plugins/style.scss [hunk 1]
```diff
@@ -160,6 +161,13 @@ $plugin-details-header-padding: 100px;
 	}
 }
 
+%placeholder {
+	cursor: default;
+	color: transparent;
+	background-color: var( --color-neutral-0 );
+	animation: loading-fade 1.6s ease-in-out infinite;
+}
+
```
Comment (cpapazoglou) [inline client/my-sites/plugins/style.scss]: TIL about the placeholder selector
https://sass-lang.com/documentation/style-rules/placeholder-selectors

**Review by cpapazoglou — COMMENTED** [review]
Works well and looks good. Addressing @WBerredo comment should be enough for merging this.

Personal taste:
I would omit the bottom border
SS 2021-11-17 at 10 57 06

File: client/my-sites/plugins/style.scss (hunk 1 above)
Comment (WBerredo) [reply client/my-sites/plugins/style.scss]: +1

Comment (gcsecsey) [reply client/my-sites/plugins/style.scss]: +1 while working on this! 😄 I didn't want to copy that same CSS into two rules, and I knew that Sass has some mechanism for this use case.

**Comment by gcsecsey** [comment]
- Added `user-selection: none` to the CSS, so the placeholder content is no longer selectable (<kbd>Cmd</kbd>+<kbd>A</kbd> won't select it either)
- Removed the horizontal separator on the placeholder
image

**Review by cpapazoglou — APPROVED** [review]
LGTM!

**Pull request merged**